TRIAGE.md

```diff
@@ -0,0 +1,49 @@
+# Triage in OpenSearch Core
```

Comment:
I'd highly recommend looking at the security team's triage guidance [1], while the format is based off communicating how the triage meeting operates the FAQ format might be useful to highlight edge cases.

- [1] https://github.com/opensearch-project/security/blob/main/TRIAGING.md